tests/realtikvtest/brietest/brie_test.go

```diff
@@ -172,7 +172,54 @@ func TestExistedTables(t *testing.T) {
 	case <-done:
 	}
 
-	for i := 0; i < 10; i++ {
+	for i := 0; i < 5; i++ {
+		tableName := fmt.Sprintf("foo%d", i)
+		tk.MustExec(fmt.Sprintf("drop table %s;", tableName))
+	}
+}
+
+func TestExistedTablesOfIncremental(t *testing.T) {
```

Comment:
please also add cases to add variation including db/table level backup and restore in sql.